packages/core/src/components/button/button.scss

```diff
@@ -209,12 +181,8 @@ button {
   }
 }
 
-:host(tds-button[disabled='true']) {
+:host([disabled]) {
   pointer-events: none;
```

Comment:
Just a thought here, isn't this also gonna disabled any hover interactions on the `tds-button`. If you have a disabled button that on hover should show a tooltip or something to highlight why it is disabled? Could be worth investigating.

Reply:
Hmm.. that is a good point and I will check it. Wonder what happens now if we attach Tooltip to it?

Reply:
I am afraid you are right @mJarsater, here is a small example with this code in: https://stackblitz.com/edit/vitejs-vite-bzyiej?file=package.json

I have no idea how I would disable clicking on Host level, I tried with onClick function that preventsDeafult and stopPropagation, but that did not work. Do you have some idea?

Reply:
I got idea, keep there...

Reply:
Maybe either expose a `tdsClick` event that you can control,or add an eventListener to the host  in some lifecycle hook?

Reply:
Code updated!

Reply:
Nice, maybe add a test to validate that this doesn't trigger a click?

Reply:
@mJarsater  - I looked into this, but could not find a proper way. 

Playwright documentation point to use `toBeDisabled`, already used in the button.e2e.ts file. Another one is to check cursor CSS or buttons CSS, which we do too.

I was thinking of implementing a small increment in the span element that exists next to the button, so if you click on the button, the number would go from 1 to 2, however, if disabling works, we should expect the number to remain the same - aka initial value.
When doing so, Playwright throws a timeout error as it can not click on the button, and the test breaks - which is expected from a logical view, but not if we consider it a mistake that breaks unit tests. 

That said, I will leave the test case as is.

Reply:
I think `toBeDisabled` might only work for native elements. I'd say what we really wanna know is if a disabled button emits a click events. Might need to "force" a click: https://playwright.dev/docs/input#forcing-the-click